app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -50,5 +50,13 @@ def textual_id
     def textual_full_name
       {:label => "Full Name", :value => @record.full_name}
     end
+
+    def textual_os_distribution
+      {:label => "Operating System Distribution", :value => @record.operating_system.try(:distribution)} if @record.operating_system.try(:distribution)
+    end
+
+    def textual_kernel_version
+      {:label => "Kernel Version", :value => @record.operating_system.try(:kernel_version)} if @record.operating_system.try(:kernel_version)
```

Comment:
Same as [this comment](https://github.com/ManageIQ/manageiq/pull/4622/files#r40979481)